src/Controller/RegionController.php

```diff
@@ -27,4 +30,24 @@ public function index(Request $request, EntityManagerInterface $entityManager):
 
         return $this->json(['status' => '200', 'message' => 'Região criada com sucesso.'], 200, ['Content-Type'=>'application/json; charset=utf-8']);
     }
+
+    #[Route('/region/{id}', name: 'korv_region_update', methods: 'PUT')]
+    public function putRegion(Request $request, EntityManagerInterface $entityManager, int $id): Response
+    {
+        $regionExists = $entityManager->getRepository(Region::class)->findOneByIdExists($id);
+        if (!$regionExists) {
+            return $this->json(['status' => '422', 'message' => 'Erro ao modificar essa região.'], 422, ['Content-Type'=>'application/json; charset=utf-8']);
```

Comment:
return $this->json(['status' => '422', 'message' => 'Erro ao modificar essa região.'], 422, ['Content-Type'=>'application/json; charset=utf-8']);